### PR TITLE
feat: add LayerZero OFT Implementation

### PR DESCRIPTION
**Pull request by larrythecucumber321**

### CODE REVIEW

File: apps/layerzero-oft/script/Bridge.s.sol [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+interface IAdapter is IOAppCore, IOFT {}
+
+contract SendOFTScript is Script {
+    using OptionsBuilder for bytes;
+
+    uint32 constant BERACHAIN_ENPOINT_ID = 40256;
+
+    // REPLACE WITH ADAPTER ADDRESS FROM STEP 1
+    address constant SEPOLIA_ADAPTER_ADDRESS =
+        0x0000000000000000000000000000000000000000;
```
Comment (itsdevbear) [inline apps/layerzero-oft/script/Bridge.s.sol:19]: Could use .Env

File: apps/layerzero-oft/script/MyOFT.s.sol [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import {Script} from "forge-std/Script.sol";
+import "../src/MyOFT.sol";
+
+contract MyOFTScript is Script {
+    address constant LAYERZERO_ENDPOINT =
+        0x6EDCE65403992e310A62460808c4b910D972f10f;
+
+    // REPLACE WITH YOUR DEPLOYED ADAPTER ON SEPOLIA
+    address constant SEPOLIA_PEER = 0x0000000000000000000000000000000000000000;
```
Comment (itsdevbear) [inline apps/layerzero-oft/script/MyOFT.s.sol:12]: Same here

File: apps/layerzero-oft/script/Bridge.s.sol (hunk 1 above)
Comment (larrythecucumber321) [reply apps/layerzero-oft/script/Bridge.s.sol:19]: Good point, thanks!

File: apps/layerzero-oft/.env.example [hunk 3]
```diff
@@ -0,0 +1,3 @@
+PRIVATE_KEY=
+SEPOLIA_ADAPTER_ADDRESS=
+BERACHAIN_OFT_ADDRESS=
```
Comment (codingwithmanny) [inline apps/layerzero-oft/.env.example:3]: Can we add the addresses here as default to make it a bit easier?

File: apps/layerzero-oft/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+- [Foundry](https://book.getfoundry.sh/getting-started/installation) - ensure `foundryup` is run to install binaries
+
+### Step 1 - Setup Project & Install Dependencies
+
+Install project dependencies:
+
+```bash
+# FROM: ./layerzero-oft
+
+pnpm install;
```
Comment (codingwithmanny) [inline apps/layerzero-oft/README.md:24]: Would this just be foundry, no need for pnpm or node in this tutorial then?

File: apps/layerzero-oft/package.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "layerzero-oft",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
```
Comment (codingwithmanny) [inline apps/layerzero-oft/package.json:10]:
```suggestion
  "author": "@larrythecucumber321",
```

File: apps/layerzero-oft/package.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "layerzero-oft",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
```
Comment (codingwithmanny) [inline apps/layerzero-oft/package.json:9]:
```suggestion
  "keywords": ["layerzero", "berachain", "bridging", "sepolia", "omnichain", "oft", "erc20"],
```

**Review by codingwithmanny — APPROVED** [review]
LGTM

**Pull request merged**